buildlib/pr/main.yml

```diff
@@ -43,6 +43,12 @@ resources:
     - container: ubuntu2004
       image: rdmz-harbor.rdmz.labs.mlnx/swx-infra/x86_64/ubuntu20.04/builder:mofed-5.0-1.0.0.0
       options: $(DOCKER_OPT_VOLUMES)
+    - container: ubuntu2204
+      image: rdmz-harbor.rdmz.labs.mlnx/swx-infra/x86_64/ubuntu20.10/builder:mofed-5.7-0.2.3.0
+      options: $(DOCKER_OPT_VOLUMES)
+    - container: ubuntu2210
+      image: rdmz-harbor.rdmz.labs.mlnx/swx-infra/x86_64/ubuntu20.10/builder:mofed-5.8-0.2.1.0
```

Comment:
is this the lowest MLNX_OFED version that supports Ubuntu 22.10?

Reply:
yes, I checked